include/eve/module/real/core/function/regular/generic/load.hpp

```diff
@@ -11,9 +11,10 @@
 #pragma once
 
 #include <eve/concept/memory.hpp>
-#include <eve/function/unsafe.hpp>
 #include <eve/detail/implementation.hpp>
 #include <eve/detail/spy.hpp>
+#include <eve/function/unsafe.hpp>
+#include <eve/wide.hpp>
```

Comment:
Normally, we include `eve/forward.hpp`, is there a reason we need the full blown wide here ?

Reply:
OK issue is concept related, will fix later